File: python/svix/models/tailscale_config.py
# this file is @generated
import typing as t

from .common import BaseModel


class TailscaleConfig(BaseModel):
    secret: str
    """Shared secret for Tailscale Webhooks"""

    timestamp_grace_seconds: t.Optional[int] = None
    """Grace period (in seconds) for the timestamp.

    If not passed, timestamp age will not be checked."""

File: kotlin/lib/src/main/kotlin/models/TailscaleConfig.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class TailscaleConfig(
    /** Shared secret for Tailscale Webhooks */
    val secret: String,
    /**
     * Grace period (in seconds) for the timestamp.
     *
     * If not passed, timestamp age will not be checked.
     */
    val timestampGraceSeconds: UInt? = null,
)
